app/assets/javascripts/discourse/app/routes/users.js

```diff
@@ -44,6 +44,7 @@ export default DiscourseRoute.extend({
 
   setupController(controller, model) {
     controller.set("columns", model.columns);
+    controller.loadGroups();
```

Comment:
I noticed `loadGroups` and `loadUsers` are both promises. `setupController` will finish before both load, which *might* be what we want. In that case the page will start to render before data is available. Once it is, those parts of the page will re-render.

An alternative would be to return `Promise.all` with both promises, so the route will wait for both to resolve before continuing.